Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -10,6 +10,7 @@ import Storage
 private let log = Logger.browserLogger
 
 private let ThumbnailIdentifier = "Thumbnail"
+private let FrecencyLimit = 20
```

Comment:
Actually, I just noticed in your code you're using 2 * max(...) with the assumption that the number of columns is always two (I see that in TopSitesLayout it's set to two) but this actually changes based on orientation https://github.com/mozilla/firefox-ios/blob/master/Client/Frontend/Home/TopSitesPanel.swift#L258